scrapi/base/helpers.py

```diff
@@ -1,8 +1,11 @@
 from __future__ import unicode_literals
 
+import re
 from copy import deepcopy
 from nameparser import HumanName
 
+URL_REGEX = re.compile(ur'(https?://\S*\/\S*)')
```

Comment:
Any reason why the middle forward slash is escaped but the first two are not?
This will also only match urls that have text after the first slash, is that expected?